src/docker-pest-command.js

```diff
@@ -3,9 +3,13 @@ const RemotePestCommand = require('./remote-pest-command');
 
 module.exports = class DockerPestCommand extends RemotePestCommand {
     get paths() {
-        let paths = {}; 
-        for (const [localPath, remotePath] of Object.entries(this.config.get("docker.paths"))) {
-            paths[ localPath.replace('${workspaceFolder}', vscode.workspace.rootPath) ] = remotePath; 
+        let paths = {};
```

Comment:
I know, not your code, but it can be changed to `const`
```suggestion
        const paths = {};
```